src/databricks/labs/ucx/hive_metastore/grants.py

```diff
@@ -80,21 +80,24 @@ def hive_grant_sql(self) -> list[str]:
         actions = self.action_type.split(", ")
         if "OWN" in actions:
             actions.remove("OWN")
-            statements.append(f"ALTER {object_type} {object_key} OWNER TO `{self.principal}`")
```

Comment:
this is out of scope of this PR, raise a separate one if you want these changes.